modules/uhttp/http.go

```diff
@@ -78,7 +78,7 @@ type Config struct {
 	modules.ModuleConfig
 	Port    int           `yaml:"port"`
 	Timeout time.Duration `yaml:"timeout"`
-	Debug   *bool         `yaml:"debug"`
+	Debug   bool          `yaml:"debug" default:"true"`
```

Comment:
Speaking of which, I think we should remove yaml tags above anyway. Nit, feel free to ignore.